src/pages/community-champions.js

```diff
@@ -1,8 +1,73 @@
 import React from 'react';
+import styled from '@emotion/styled';
 import Layout from '~components/dev-hub/layout.js';
+import HeroBanner from '~components/dev-hub/hero-banner';
+import { P, H2 } from '~components/dev-hub/text';
+import Button from '~components/dev-hub/button';
+import { screenSize, size } from '~components/dev-hub/theme';
+import BannerImage from '~images/community-champions/community-champions-banner-icon.png';
+
+const APPLY_BUTTON_BOTTOM_MARGIN = '14px';
+const APPLY_BUTTON_MOBILE_BOTTOM_MARGIN = '12px';
+
+const Subtitle = styled(P)`
+    color: ${({ theme }) => theme.colorMap.greyLightTwo};
+    margin-top: ${size.mediumLarge};
+    @media ${screenSize.upToLarge} {
+        margin-top: ${size.default};
+    }
+`;
+
+const ApplyButton = styled(Button)`
+    margin-top: ${size.large};
+    margin-bottom: ${APPLY_BUTTON_BOTTOM_MARGIN};
+    @media ${screenSize.upToLarge} {
+        margin-top: ${size.default};
+        margin-bottom: ${APPLY_BUTTON_MOBILE_BOTTOM_MARGIN};
+    }
+`;
+
+const StyledHeroBanner = styled(HeroBanner)`
+    > div {
+        background-size: auto;
+    }
+`;
 
 const CommunityChampions = () => {
-    return <Layout />;
+    const communityChampionBreadcrumbs = [
```

Comment:
This is awesome as it sets up the breadcrumbs for navigation, one small performance improvement would be to declare this constant outside of the component.

We would want to do this because whenever this component renders (or re-renders), this is one less action we would have to do (load again the same constant that does not require any `props` or anything). This is a super super small improvement however on other components it can make much more of an impact, which is why I am pointing it out here. Otherwise this is great!